deploy/operator/clickhouse-operator-install-bundle-v1beta1.yaml

```diff
@@ -3344,7 +3344,6 @@ metadata:
   labels:
     clickhouse.altinity.com/chop: 0.22.2
     app: clickhouse-operator
-data:
```

Comment:
why? which effect?

Reply:
@Slach thanks for clarifying. Is not an actual issue, but the property looks to be empty (nothing declared inside).
Does not cause problems other than DX (get false warning for linters or that kind of stuff).